tools/fix-manpages

```diff
@@ -0,0 +1,6 @@
+/.SS "Usage:"/,+3d
+/.SH NOTES\|.SH TIME SPECIFICATION/{ n
+N
+N
+d
+}
```

Comment:
Add comments about what this is doing and why

Reply:
What, you don't think we're going to remember what these magic sed commands do?